README.md

```diff
@@ -23,6 +23,7 @@ We created hummingbot to promote **decentralized market-making**: enabling membe
 | <img src="documentation/docs/assets/img/kucoin_logo.png" alt="KuCoin" width="90" /> | kucoin | [KuCoin](https://www.kucoin.com/) | 1 | [API](https://docs.kucoin.com/#general) |
```

Comment:
Should the Bittrex api be updated to 3? @dennisocana

Reply:
@carlolm yes, good catch. I will make the change now.